Source/Particles/LaserParticleContainer.cpp

```diff
@@ -159,6 +160,7 @@ LaserParticleContainer::LaserParticleContainer (AmrCore* amr_core, int ispecies,
         m_position[0] += (Z0_boost-m_Z0_lab)*m_nvec[0];
         m_position[1] += (Z0_boost-m_Z0_lab)*m_nvec[1];
         m_position[2] += (Z0_boost-m_Z0_lab)*m_nvec[2];
+        m_vz_antenna = (m_vz_antenna-WarpX::beta_boost)/(1._rt-WarpX::beta_boost*m_vz_antenna);
```

Comment:
```suggestion
```
It seems that this leads to problems e.g. in the case of a boosted-frame simulation with (user-input) `vz_antenna = 0`. In this case, `m_vz_antenna` (as given by the above formula) is non-zero. In this case:
- There is double-counting in the motion of the antenna: the `z` position gets updated twice, because of these code snippets:
```
            // When running in the boosted-frame, their is additional velocity along nvec
            if (gamma_boost > 1.){
                vx -= PhysConst::c * beta_boost * tmp_nvec_0;
                vy -= PhysConst::c * beta_boost * tmp_nvec_1;
                vz -= PhysConst::c * beta_boost * tmp_nvec_2;
            }
```
```
z += vz * dt + vz_antenna * PhysConst::c * dt;
```
- In addition, the field amplitude gets modified because `vel_dot` is non-zero in the code below
```
m_mobility = m_mobility/WarpX::gamma_boost * (1._rt - vel_dot);
```

For now, I would suggest that moving antenna and boosted-frame are two options that cannot be combined. (This is also what we do in FBPIC, I think.)